jscomp/others/js_promise.ml

```diff
@@ -63,6 +63,9 @@ external all4 : 'a0 t * 'a1 t * 'a2 t * 'a3 t -> ('a0 * 'a1 * 'a2 * 'a3) t
   = "all"
   [@@bs.val] [@@bs.scope "Promise"]
 
+external then2 : 'a t -> (('a -> 'b t)[@bs.uncurry]) -> 'b t = "then"
```

Comment:
remove this, that's needed for examples
a t-first version for promises should be used